diff --git a/client/filters/filter_v1_post_parameters.go b/client/filters/filter_v1_post_parameters.go
index 1d42920..e78276f 100644
--- a/client/filters/filter_v1_post_parameters.go
+++ b/client/filters/filter_v1_post_parameters.go
@@ -68,7 +68,7 @@ type FilterV1PostParams struct {
 
 	Sample: home, public
 	*/
-	Context []string
+	FContext []string
 
 	/* ExpiresIn.
 
@@ -170,14 +170,14 @@ func (o *FilterV1PostParams) SetHTTPClient(client *http.Client) {
 }
 
 // WithContext adds the context to the filter v1 post params
-func (o *FilterV1PostParams) WithContext(context []string) *FilterV1PostParams {
-	o.SetContext(context)
+func (o *FilterV1PostParams) WithFContext(context []string) *FilterV1PostParams {
+	o.SetFContext(context)
 	return o
 }
 
 // SetContext adds the context to the filter v1 post params
-func (o *FilterV1PostParams) SetContext(context []string) {
-	o.Context = context
+func (o *FilterV1PostParams) SetFContext(context []string) {
+	o.FContext = context
 }
 
 // WithExpiresIn adds the expiresIn to the filter v1 post params
@@ -232,7 +232,7 @@ func (o *FilterV1PostParams) WriteToRequest(r runtime.ClientRequest, reg strfmt.
 	}
 	var res []error
 
-	if o.Context != nil {
+	if o.FContext != nil {
 
 		// binding items for context[]
 		joinedContext := o.bindParamContext(reg)
@@ -305,7 +305,7 @@ func (o *FilterV1PostParams) WriteToRequest(r runtime.ClientRequest, reg strfmt.
 
 // bindParamFilterV1Post binds the parameter context[]
 func (o *FilterV1PostParams) bindParamContext(formats strfmt.Registry) []string {
-	contextIR := o.Context
+	contextIR := o.FContext
 
 	var contextIC []string
 	for _, contextIIR := range contextIR { // explode []string
diff --git a/client/filters/filter_v1_put_parameters.go b/client/filters/filter_v1_put_parameters.go
index 66d7a1f..ebf23b5 100644
--- a/client/filters/filter_v1_put_parameters.go
+++ b/client/filters/filter_v1_put_parameters.go
@@ -68,7 +68,7 @@ type FilterV1PutParams struct {
 
 	Sample: home, public
 	*/
-	Context []string
+	FContext []string
 
 	/* ExpiresIn.
 
@@ -176,14 +176,14 @@ func (o *FilterV1PutParams) SetHTTPClient(client *http.Client) {
 }
 
 // WithContext adds the context to the filter v1 put params
-func (o *FilterV1PutParams) WithContext(context []string) *FilterV1PutParams {
-	o.SetContext(context)
+func (o *FilterV1PutParams) WithFContext(context []string) *FilterV1PutParams {
+	o.SetFContext(context)
 	return o
 }
 
 // SetContext adds the context to the filter v1 put params
-func (o *FilterV1PutParams) SetContext(context []string) {
-	o.Context = context
+func (o *FilterV1PutParams) SetFContext(context []string) {
+	o.FContext = context
 }
 
 // WithExpiresIn adds the expiresIn to the filter v1 put params
@@ -249,7 +249,7 @@ func (o *FilterV1PutParams) WriteToRequest(r runtime.ClientRequest, reg strfmt.R
 	}
 	var res []error
 
-	if o.Context != nil {
+	if o.FContext != nil {
 
 		// binding items for context[]
 		joinedContext := o.bindParamContext(reg)
@@ -327,7 +327,7 @@ func (o *FilterV1PutParams) WriteToRequest(r runtime.ClientRequest, reg strfmt.R
 
 // bindParamFilterV1Put binds the parameter context[]
 func (o *FilterV1PutParams) bindParamContext(formats strfmt.Registry) []string {
-	contextIR := o.Context
+	contextIR := o.FContext
 
 	var contextIC []string
 	for _, contextIIR := range contextIR { // explode []string
